### DISCUSSION TITLE
Vercel example

### DISCUSSION THREAD

**Discussion by Adam-Collier** [Josh-McFarlin/remix-image#16] [Q&A] [answered]
Hey Josh 👋, great work on the library! I was wondering if you'd be able to add a Vercel example? I saw that you've created a Cloudflare example and I think Vercel would be a great addition (especially for those who have moved away from Next but love Vercels DX)

**Comment by Josh-McFarlin** [comment, marked as answer]
Hi Adam,

Thanks for the suggestion! I created a Vercel example:

- Code: [/examples/vercel](https://github.com/Josh-McFarlin/remix-image/tree/master/examples/vercel)
- Deployment: https://remix-image-vercel.vercel.app/

Let me know if you have any issues deploying your own project!

**Comment by jorisre** [comment]
Hi @Josh-McFarlin 

Thank you for the library, I just have a question.

Use of `os.tempdir()` means cache isn't persisted between deployments, right?

👉🏻 https://github.com/Josh-McFarlin/remix-image/blob/master/examples/vercel/app/routes/api/image.ts#L32
